src/syscalls/deprecated_business_logic_syscall_handler.rs

```diff
@@ -298,6 +298,9 @@ impl<'a, S: StateReader, C: ContractClassCache> DeprecatedBLSyscallHandler<'a, S
         &mut self,
         vm: &VirtualMachine,
         syscall_ptr: Relocatable,
+        #[cfg(feature = "cairo-native")] _program_cache: Option<
+            Rc<RefCell<ProgramCache<'_, ClassHash>>>,
+        >,
```

Comment:
Does it make sense to include this? The deprecated syscall handler will not be used when running native (aka cairo 1 contracts compiled to sierra)